docs/update-the-search-index.md

```diff
@@ -0,0 +1,25 @@
+# Update the Search Index
+
+To update the search index (likely after a change to the documentation), you need to run the following command, adding the relevant values from [ownCloud’s Algolia account](https://www.algolia.com/apps/RQPV9Q61S4/api-keys/all).
+
+**Note:** if you do not have access to ownCloud’s Algolia account, please contact tboerger@owncloud.com.
+```
+docker run \
+    -v $(pwd)/docsearch-scraper:/opt/docsearch-scraper:ro \
```

Comment:
Where does docker run? If locally, something needs to be added as installation prerequisite...

Reply:
I've made an update which should solve this issue.